scripts/npm/update-bpk-dependencies.js

```diff
@@ -0,0 +1,73 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ /* eslint-disable no-console */
+
+const { execSync } = require('child_process');
+
+const updatedFiles = [];
+
+const updateBpkDependencies = (packageFile, errorIfOutdated) => {
+  const command = `ncu '/^(react-native-)?bpk-.*$/' -a -u --packageFile ${packageFile}`;
+  const result = execSync(command).toString();
+  if (!result.includes('All dependencies match the latest')) {
+    console.log('');
+    console.log(`Outdated bpk package dependencies in ${packageFile}! 😱`);
+    console.log('');
+    updatedFiles.push(packageFile);
+    if (errorIfOutdated) {
+      process.exit(1);
+    }
+  }
+};
+
+const installNcu = () => {
+  try {
+    execSync('npm list -g npm-check-updates');
+  } catch (e) {
+    console.log('NCU not installed 😱');
+    console.log('Installing `npm-check-updates` now! 🙉 ');
+    execSync('npm install -g npm-check-updates');
```

Comment:
This is naughty! Can we find a solution that doesn't rely on a third party tool - especially one that we have to install globally? (don't let hugo see this 😂 )

Reply:
Perhaps something along the lines of:

1. grab a list of `package.json`'s
2. create a map of `"package-name": version,`
3. iterate through each package and check `dependencies` & `devDependencies` against package version map
4. if they don't match, throw error

Reply:
I thought about that, but I didn't like the manual feeling of it.
Happy to go down that route instead if you think it's better